src/DI/OrmCacheExtension.php

```diff
@@ -119,7 +119,7 @@ protected function getDriverCache(string $service): ServiceDefinition
 			->setFactory($config['driver']);
 
 		if (is_subclass_of($config['driver'], FilesystemCache::class)) {
-			$driverCache->setArguments([$builder->expand('%tempDir%/cache/Doctrine.' . ucfirst($service))]);
+			$driverCache->setArguments([$builder->parameters['tempDir'] . '/cache/Doctrine.' . ucfirst($service)]);
```

Comment:
Could you please switch to Helpers::expand?

Reply:
imo it's unnecessary. You only need parameter from ContainerBuilder and it's more faster. But you're boss :) Please confirm again, if you want to change it.

Reply:
Well, you're right. My intention was to keep things same. But it's OK, let it be. :-)